companies.md

```diff
@@ -8,15 +8,15 @@ Go [Home](/). Jump to: <a href="/events.html">Events</a>. Jump to: <a href="/uni
 
 <a name="companies"></a>
 ## Companies - {{ site.data.companies | size }}
-{% assign sorted = site.data.companies | sort_natural: 'name' %}
+{% assign sorted = site.data.companies | sort %}
```

Comment:
If `sort_natural` still works, we'd probably want to keep that.

Reply:
I'll give it a shot. Still getting Ruby set up locally.

Reply:
![image](https://user-images.githubusercontent.com/7773256/76673750-3bad9780-6576-11ea-9082-2ebc042329f3.png)
Looks like `sort_natural` does not work.

Reply:
Yep, i had found that one. 
 Addressed in https://github.com/phildini/stayinghomeclub/pull/864/commits/b3ea5ead5744d06fac0c19cdfe431a194c6f8bbb.